src/logic/quickstart.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  META_FILE,
+  NormalizedProviderSettings,
+  SuperJson,
+} from '@superfaceai/one-sdk';
+import { exec } from 'child_process';
+import { join as joinPath } from 'path';
+import { promisify } from 'util';
+
+import { LogCallback } from '../common/log';
+
+const execShell = promisify(exec);
+
+export async function installSdk(options?: {
+  logCb?: LogCallback;
+  warnCb?: LogCallback;
+}): Promise<void> {
+  //TODO: check for yarn/package lock and use yarn/npm - npm/yarn init if we dont find package.json? Wher should we look?
```

Comment:
Imo it's enough to run `yarn add @superfaceai/one-sdk` or `npm install @superfaceai/one-sdk`.

As to yarn/npm detection, we could either implement it by going up the directories until we bump into package.json and then look for yarn.lock

or we can look for yarn.lock in the current directory if we don't find it run npm. Both yarn and npm seem to do the package.json search themselves, so the worst case this can and up with is calling npm install inside a child directory of a yarn project.